src/resources/js/components/Presentation.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="container">
+     <div v-if="isError" class="error">
+          <h3>{{ message }}</h3>
+     </div>
+
+     <h3>Presentation info</h3>
+     <div>
+          <h4>title : {{ presentation.title }} </h4>
+          <p>created on : {{ formatDate(presentation.date) }} </p>
+     </div>
+
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+  name: "Presentation",
+  props: ["idPresentation"], // we will send the id of the presentation through props
+  data() {
+    return {
+      presentation: {},
+      polls: [],
+      isError: false,
+      message: ''
+    };
+  },
+  beforeMount() {
+    this.getPresentationInfo();
+  },
+  methods: {
+     getPresentationInfo() {
+      //const id = idPresentation;
+      const id = 1;
+      const apiUrl = "api/v1/presentations/" + id;
+
+      axios
+        .get(apiUrl)
+        .then(response => {
+          this.presentation = response.data.data;
+        })
+        .catch(error => {
+          this.isError = true;
+          this.message = "an error has occured while sending the data";
+        });
+    },
+    formatDate(dateString){
+         /**
+          * formatting date
+          * https://codehandbook.org/javascript-date-format/
+          */
+         let currentDate = new Date(dateString);
+         let formatted_date = currentDate.getFullYear() + "-" + (currentDate.getMonth() + 1) + "-" + currentDate.getDate();
+         return formatted_date;
```

Comment:
you should use moment.js library for date formating

Reply:
I created the issue #120 and will install this module there, I have removed the date formatting of this PR in [cb88d](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/113/commits/cb88d6199852122ae6014f1528d295a703e1d824)